New marketing URL per INC12462054

Adam42 opened the pull request.

Adam42 committed:
New marketing URL per INC12462054

dsmk reviewed (changes requested):
Adam, I did not see an entry in the sites.map to correspond to the redirects.map file.  The redirects.map is only consulted if the sites.map indicates a redirect or a redirect_asis backend.   See the SOP on webrouter configuration for more information.

I believe that you want the following in sites.map:
_/grandopening redirect_asis ;

Because the destination of the redirect appears to be an absolute URL and you do not want NGINX to add the original request_uri (/grandopening) to the end of the request.

Adam42 committed:
Add site entry for redirect/marketing URL

Previous commit only included an entry addition to redirect map, we need this entry to have the webrouter look for a redirect for this URL.

Adam42 commented:
Thanks for catching that @dsmk I've added the corresponding entry to sites.map. It is indeed a redirect_asis and the original request_uri should not be appended.
